src/test/resources/vm-publish-consume-config.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd">
+
+    <flow name="listener">
+        <vm:listener queueName="queue" queueType="TRANSIENT">
+            <vm:response>
+                <vm:body>
```

Comment:
we are using `content` for `publish` and `publishConsume` but `body` for `response`, better if we pick one since they are the same thing.